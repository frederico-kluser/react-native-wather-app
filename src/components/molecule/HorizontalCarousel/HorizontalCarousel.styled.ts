import styled from 'styled-components/native';

const Container = styled.View``;

export const Title = styled.Text`
  font-size: 17px;
`;

export default Container;
