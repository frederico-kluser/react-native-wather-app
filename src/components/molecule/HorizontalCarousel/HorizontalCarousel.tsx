import React from 'react';
import Container, {Title} from './HorizontalCarousel.styled';

const HorizontalCarousel = ({}: any) => (
  <Container>
    <Title>Today</Title>
  </Container>
);

export default HorizontalCarousel;
