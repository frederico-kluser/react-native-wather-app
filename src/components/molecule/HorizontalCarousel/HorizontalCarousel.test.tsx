import React from 'react';
import renderer from 'react-test-renderer';
import HorizontalCarousel from './HorizontalCarousel';

test('renders correctly', () => {
  const tree = renderer.create(<HorizontalCarousel />).toJSON();
  expect(tree).toMatchSnapshot();
});
