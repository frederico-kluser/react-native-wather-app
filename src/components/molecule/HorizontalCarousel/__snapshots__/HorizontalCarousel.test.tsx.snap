// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {},
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "fontSize": 17,
        },
      ]
    }
  >
    Today
  </Text>
</View>
`;
